docs/core/tutorials/using-on-mac-vs-full-solution.md

```diff
@@ -188,3 +188,4 @@ Unit tests provide automated software testing during your development and publis
 ## See also
 
 - [Visual Studio 2017 for Mac Release Notes](/visualstudio/releasenotes/vs2017-mac-relnotes)
+- [Run unit tests with Test Explorer](visualstudio/test/run-unit-tests-with-test-explorer)
```

Comment:
This topic is specific to Visual Studio on Windows and the OP was asking about VS for Mac, so I'm not sure this fixes the issue. @thraka?

Reply:
May be add corresponding topic in VS for Mac docs first?

Reply:
Consider adding the corresponding topic in VS for Mac docs first?